### PR TITLE
Latent transformation draft2

### PR DESCRIPTION
**Pull request by ajlee21**
In this PR:

1.  Scripts modified to be are reproducible -- added seeds where needed
2.  Script added to identify genes that are better modeled by the latent space vs linear space transformation
3.  Files are organized by dataset used
4.  Scripts re-run to and new intermediate and output files added

### CODE REVIEW

File: scripts/nbconverted/generate_input_continuum.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+import seaborn as sns
+import matplotlib.pyplot as plt
+np.random.seed(123)
+
+
+# In[3]:
+
+
+# load arguments
+#data_file = '/home/alexandra/Documents/Pseudomonas_scratch/all-pseudomonas-gene-normalized.pcl'
```
Comment (huqiwen0313) [inline scripts/nbconverted/generate_input_continuum.py]: Delete this line ? (you comment out)

File: scripts/nbconverted/generate_input_continuum.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+# In[9]:
+
+
+# Output training and test sets
+
+# training data
+maxO2.to_csv(os.path.join(os.path.dirname(os.getcwd()), "data", "train_maxO2.txt"), sep='\t')
+minO2.to_csv(os.path.join(os.path.dirname(os.getcwd()), "data", "train_minO2.txt"), sep='\t')
+#input_holdout.to_csv(os.path.join(os.path.dirname(os.getcwd()), "data", "train_model_input_anr.txt.xz"), sep='\t', compression='xz')
+
```
Comment (huqiwen0313) [inline scripts/nbconverted/generate_input_continuum.py]: also this one.

File: scripts/nbconverted/generate_input_bool.py [hunk 3]
```diff
@@ -134,21 +136,23 @@
...
-fig_file = os.path.join(os.path.dirname(os.getcwd()), "viz", "cv.png")
-fig.savefig(fig_file)
+#fig_file = os.path.join(os.path.dirname(os.getcwd()), "viz", "cv.png")
+#fig.savefig(fig_file)
 
 #train_control.to_csv(os.path.join(os.path.dirname(os.getcwd()), "data", "train_control.txt"), sep='\t')
 #train_treat.to_csv(os.path.join(os.path.dirname(os.getcwd()), "data", "train_treat.txt"), sep='\t')
 
 #test_control.to_csv(os.path.join(os.path.dirname(os.getcwd()), "data", "test_control.txt"), sep='\t')
 #test_treat.to_csv(os.path.join(os.path.dirname(os.getcwd()), "data", "test_treat.txt"), sep='\t')
```
Comment (huqiwen0313) [inline scripts/nbconverted/generate_input_bool.py]: clean up these commented out lines ?

**Review by huqiwen0313 — APPROVED** [review]
Looks good to me ! Just some clean ups.

**Comment by ajlee21** [comment]
So the lines you mentioned are all clean in the versions from the commit from Aug 1 "scripts to redo cipro treatment analysis"

I"m sorry there were a lot of commits so its not a surprise that files were mixed up

**Pull request merged**